### ISSUE TITLE
Can't commit

### ISSUE THREAD

**Issue by BenevidesLecontes** [esvit/ng-table#949]
HI! Thanks for your amazing work, i forked your repo and i did a couple of changes, but when i do commit or npm run cm, it show the commitezen messages, i fill all the questions and after that it throws so many errors and i can't commit my changes to my github because of this. Am i missing something? Here's some errors some random message.

<img width="652" alt="screen shot 2016-12-21 at 18 21 55" src="https://cloud.githubusercontent.com/assets/1839361/21404909/8b88d28c-c7aa-11e6-9129-0aebff48953b.png">

**Comment by christianacca** [comment]
Looks like the pre-commit hook is stopping the commit (a good thing in this
case). Specifically, it looks like you're trying to commit typescript code
that has compile errors.

**Comment by BenevidesLecontes** [comment]
Yes i saw that, but i didn't changed nothing on the code, i only forked your repo and removed bundles from git ignore, to use the repo in my private bower.
